src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -159,20 +173,6 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Include="@(NativeFramework->'-framework %(Identity)')" Condition="'$(TargetOS)' == 'osx'" />
     </ItemGroup>
 
-    <Exec Command="command -v &quot;$(CppLinker)&quot;" IgnoreExitCode="true" StandardOutputImportance="Low">
-      <Output TaskParameter="ExitCode" PropertyName="_WhereLinker" />
-    </Exec>
-
-    <Exec Command="command -v &quot;$(CppCompilerAndLinkerAlternative)&quot;" Condition="'$(CppCompilerAndLinkerAlternative)' != '' and '$(_WhereLinker)' != '0'" IgnoreExitCode="true" StandardOutputImportance="Low">
-      <Output TaskParameter="ExitCode" PropertyName="_WhereLinkerAlt" />
-    </Exec>
-
-    <PropertyGroup Condition="'$(CppCompilerAndLinkerAlternative)' != '' and '$(_WhereLinker)' != '0' and '$(_WhereLinkerAlt)' == '0'">
```

Comment:
I think we should keep the compiler detection code as it helps in build containers. We just need to make sure that we cross-compile only on the machines with clang and skip it when clang is not found.

Reply:
Ah, I can see it was moved above. 👍